packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1270,3 +1266,29 @@ export default decorate([
     )
   },
 ])
+
+export default decorate([
+  addSubscriptions({
+    remotes: subscribeRemotes,
+  }),
+  provideState({
+    initialState: () => ({ suggestedExcludedTags: undefined }),
```

Comment:
`suggestedExcludedTags` it's not a state prop which can be changed by a user, so, it can be a computed.